openapi/commercetools/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Commerce/eCommerce", "Cost/Paid"]
+org = "ballerinax"
+name = "commercetools"
+icon = "icon.png"
+distribution = "2201.0.0"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "1.0.0"
```

Comment:
@SanduDS As we have a requirement to release this connector with SLBeta6, lets use `1.0.0` for SLBeta6 and `1.1.0` for `2201.0.0`

Also please send a PR to https://github.com/ballerina-platform/openapi-connectors/tree/slbeta6-batch-4 (slbeta6-batch-4 branch with necessary changes for SLBeta6)